python/src/deltachat/const.py

```diff
@@ -5,3 +5,8 @@
     if name.startswith("DC_"):
         globals()[name] = getattr(lib, name)
 del name
+
+DC_IMEX_EXPORT_SELF_KEYS = 1
+DC_IMEX_IMPORT_SELF_KEYS = 2
+DC_IMEX_EXPORT_BACKUP = 11
+DC_IMEX_IMPORT_BACKUP = 12
```

Comment:
For some reason all the other DC_ constants are created automatically, but these four not. Do you have any idea why?

Reply:
Add `DC_IMEX` to this regexp: https://github.com/deltachat/deltachat-core-rust/blob/5535475cc9951134b3b735b51460b458c57fab86/python/src/deltachat/_build.py#L138

Reply:
Thanks